platform/ios/src/MGLMapView.h

```diff
@@ -1079,6 +1079,20 @@ IB_DESIGNABLE
 
 #pragma mark - Runtime styling API
 
+/**
+ Returns an instance of `MGLStyle` for the current map style.
+ 
+ The returned object is a proxy for the current map style for customization 
+ purposes. It also has a set of convenience methods for creating style URLs of 
+ default styles provided by Mapbox.
+ 
+ @note The default styles provided by Mapbox contain sources and layers with
+    identifiers that will change over time. Applications that use APIs that 
+    manipulate a style's sources and layers must first set the style URL to an 
+    explicitly versioned style using a convenience method like
+    `+[MGLStyle outdoorsStyleURLWithVersion:]`, `MGLMapView`'s “Style URL”
+    inspectable in Interface Builder, or a manually constructed `NSURL`.
+ */
```

Comment:
Please update the macOS implementation of MGLMapView with these changes as well.